packages/pigeon/run_tests.sh

```diff
@@ -298,7 +298,7 @@ run_android_unittests() {
   gen_android_unittests_code ./pigeons/void_arg_host.dart VoidArgHost
   gen_android_unittests_code ./pigeons/voidflutter.dart VoidFlutter
   gen_android_unittests_code ./pigeons/voidhost.dart VoidHost
-  cd platform_tests/android_unit_tests
+  cd platform_tests/alternate_language_test_plugin/example
```

Comment:
> The idea is that we'll move to the standard plugin testing scripts at some point, not here, right?

Ideally; see PR description.

> Will @tarrinneal integration tests get run here?

These scaffolds will host the integration tests as well; running them will be a different step since it will use `flutter drive` rather than `gradle`.